api/src/main/java/io/minio/BucketArgs.java

```diff
@@ -39,9 +39,13 @@ public abstract static class Builder<B extends Builder<B, A>, A extends BucketAr
       extends BaseArgs.Builder<B, A> {
     private static final Pattern BUCKET_NAME_REGEX =
         Pattern.compile("^[a-z0-9][a-z0-9\\.\\-]{1,61}[a-z0-9]$");
+    private boolean validateBucket = true;
```

Comment:
1. rename this member as `private boolean skipValidation = false;` and fix it accordingly
2. Add skip validation check to `ObjectArgs` as well.